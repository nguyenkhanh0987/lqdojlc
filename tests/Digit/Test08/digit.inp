100
98137 99628
3655 33379
1542 95629
46387 78282
77349 95819
45852 84765
40032 89691
59322 68934
98389 99869
92986 98717
46421 71845
50220 94274
58119 70667
21659 97451
35143 38637
79711 83261
72540 73151
15676 50841
46367 61300
96369 97354
74451 82558
4054 61383
66171 83931
31083 60920
18056 41536
65429 95495
26168 73895
36939 55188
17694 97132
67506 98316
9729 99562
65049 83322
63040 67536
32037 38265
86252 92947
83601 99310
29857 54077
70605 84530
92057 98426
19947 93922
29790 99724
30114 73061
28482 49093
23425 90959
99148 99319
85933 91052
67713 94414
89222 89655
93953 98704
69381 76757
51934 84510
6250 77309
80762 98082
33132 95909
44166 84826
17681 27425
40247 93307
51899 77045
53737 76746
8992 25981
14938 96950
41837 70291
84467 98240
56586 78648
39241 52102
77885 81584
60916 97286
59350 83999
40449 86900
53770 90772
77432 77721
36961 45966
31411 39763
98538 99713
47493 52897
19405 41122
86076 99135
3694 5051
5854 97118
44370 70765
89006 97060
11468 61619
98583 99725
12148 97869
3303 23590
98455 99383
13681 30985
79228 96220
30842 75565
48646 78552
95980 98794
68582 83838
9382 19631
97528 99760
79052 99177
32647 87982
80041 91582
60250 92131
24454 78431
89898 92336
