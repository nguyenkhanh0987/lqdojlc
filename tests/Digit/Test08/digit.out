124
5548
10649
2567
676
2889
3560
608
97
163
2256
2443
1035
6610
689
22
8
5374
1593
38
452
8727
1007
3371
3706
1425
5098
2062
7371
1352
8882
1097
313
888
277
568
2651
535
155
6637
5238
4090
2714
6342
31
302
1206
53
159
233
2082
9082
476
4826
3054
1668
3557
1715
1639
2920
8046
2655
466
1580
1351
257
1634
1301
3289
2341
16
1085
1283
88
557
3250
482
247
9913
2490
174
6899
87
8488
4038
74
3252
495
4159
2200
133
719
2053
147
691
4623
361
1545
5677
1
